src/widgets.js

```diff
@@ -267,6 +267,12 @@ define([
             return widget.input.val();
         };
 
+        mug.on("property-changed", function (e) {
```

Comment:
Oh. I didn't realize it was "legacy". I don't think I know enough about the context of this one to suggest another option either. 

> I think I did a similar thing elsewhere in this module.

Yep. that's why I used this (link in my last comment)